source/slang/CMakeLists.txt

```diff
@@ -274,6 +274,10 @@ slang_add_target(
     PUBLIC_HEADERS ${slang_SOURCE_DIR}/slang*.h ${CMAKE_CURRENT_BINARY_DIR}/slang-version-header/*.h
 )
 
+if(CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang")
```

Comment:
Is this options specific to `AppleClang` or would just `Clang` be more appropriate?

Reply:
I get the warning on Linux with regular clang

![image](https://github.com/user-attachments/assets/5abe0db4-5741-4056-adf0-e737b28bd0ee)

Reply:
Interesting.
I will make it to AppleClang and Clang.
```
if(CMAKE_CXX_COMPILER_ID STREQUAL "AppleClang" OR CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
```